tests/openkit-shared-tests/Protocol/BeaconTest.cs

```diff
@@ -1126,5 +1128,85 @@ public void NamedEventReportedForDataCollectionLevel2()
             //then
             Assert.That(target.IsEmpty, Is.False);
         }
+
+        [Test]
+        public void SessionStartIsReported()

```

Comment:
Be aware that this test calls `StartSession` twice. Ones implicitly from the Session constructor and once explicitly.